canonical/vanilla-framework
# Percy examples combination - `base/code`

Combines `base/code` examples. 

Inline & block were moved into a new `base/code` directory so that they could be combined in `base/code/combined`. So I also added redirects to keep their old links supported.

Depends on #5142 (includes its code)

## QA

- Review [base code combined examples](https://vanilla-framework-5143.demos.haus/docs/examples/base/code/combined?theme=light)
- Review [base code inline docs](https://vanilla-framework-5143.demos.haus/docs/base/code#inline); they have been updated to remove the old dark mode code-inline-dark embed.
- Verify redirects
    - https://vanilla-framework-5143.demos.haus/docs/examples/base/code-block redirects to https://vanilla-framework-5143.demos.haus/docs/examples/base/code/block
    - https://vanilla-framework-5143.demos.haus/docs/examples/base/code-inline-dark redirects to https://vanilla-framework-5143.demos.haus/docs/examples/base/code/inline?theme=dark
    - https://vanilla-framework-5143.demos.haus/docs/examples/base/code-inline redirects to https://vanilla-framework-5143.demos.haus/docs/examples/base/code/inline
 - Make sure the embedded code docs are not missing / failing to embed due to the link path changing
